### DISCUSSION TITLE
Having error while deploying the contract

### DISCUSSION THREAD

**Discussion by prakharmishra3956** [Cyfrin/foundry-full-course-cu#2620] [Q&A] [answered]
having this error, can't find the solution. Please help!!
![Screenshot (29)](https://github.com/user-attachments/assets/fac150c0-f496-4678-9969-8daece8b9fe6)

**Comment by swaindhruti** [comment, marked as answer]
As the error indicates an issue with contract source formatting, please try the following command: `forge script script/DeploySimpleStorage.s.sol:Script`.

**Reply by prakharmishra3956** [reply]
Hey thanks, it's compiling but the error still persists.
" Error: 
Could not find target contract "

**Reply by EngrPips** [reply]
If the codebase is compiling fine you can ignore the error.
